specification/search/resource-manager/Microsoft.Search/stable/2025-05-01/search.json

```diff
@@ -621,6 +621,60 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Search/searchServices/{searchServiceName}/upgrade": {
```

Comment:
do we need a reference to the systemData definition as par tof the SearchService Model?

        "systemData": {
          "readOnly": true,
          "description": "Azure Resource Manager metadata of the search service containing createdBy and modifiedBy information.",
          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/systemData"
        }